pkg/bot/discord.go

```diff
@@ -253,22 +276,21 @@ func (b *Discord) handleMessage(ctx context.Context, dm discordMessage) error {
 	})
 
 	response := e.Execute(ctx)
-	err := b.send(dm.Event, req, response)
+	err := b.send(dm.Event.ChannelID, response)
 	if err != nil {
 		return fmt.Errorf("while sending message: %w", err)
 	}
 
 	return nil
 }
 
-func (b *Discord) send(event *discordgo.MessageCreate, req string, resp interactive.Message) error {
-	b.log.Debugf("Discord incoming Request: %s", req)
+func (b *Discord) send(channelID string, resp interactive.Message) error {
 	b.log.Debugf("Discord Response: %s", resp)
 
 	markdown := interactive.RenderMessage(b.mdFormatter, resp)
 
 	if len(markdown) == 0 {
-		return fmt.Errorf("while reading Slack response: empty response for request %q", req)
```

Comment:
So, we used to know which request caused an empty response, now we don't have it. It would be valuable for debug purposes, do we have it in other places and remove it here, or completely lost it?

Reply:
The problem is that in case of automation we don't know the actual request. As I use the `send` method for it as well, I moved the debug logging for the request in the `handleMessage` method:

```go
	b.log.Debugf("Slack incoming Request: %s", request)
```

but yeah, with such approach don't have the context for a given error. 
I can bring it back though - I'll still pass `request` to the `send` method, and in case of automation I will set it to e.g.  `Automation`. WDYT?

Reply:
> The problem is that in case of automation we don't know the actual request. As I use the `send` method for it as well, I moved the debug logging for the request in the `handleMessage` method:
> 
> ```go
> 	b.log.Debugf("Slack incoming Request: %s", request)
> ```
> 
> but yeah, with such approach don't have the context for a given error. I can bring it back though - I'll still pass `request` to the `send` method, and in case of automation I will set it to e.g. `Automation`. WDYT?

Now it is clear, I think we can keep what you did 🆗 Thanks